docs/api-reference/localization/overview.md

```diff
@@ -9,12 +9,12 @@ By default the first language is English `en`. The default language can be chang
 ```js
 import {combineReducers} from 'redux';
 import keplerGlReducer from 'kepler.gl/reducers';
-import {LOCALES} from 'kepler.gl/constants';
+import { LOCALE_CODES } from 'kepler.gl/localization/locales';
```

Comment:
Please set your editor's prettier `bracketSpacing` to be `false`

Reply:
For this import to work, we will need to add `localization.js` to package.json's `files: []` property. Otherwise, it is not exposed to be imported.

You should also `export {LOCALE_CODES}`  in the `localization/index.js`, instead of using an additional path. which will not be accessible when the library is built.

Reply:
Thanks @heshan0131 I have made the import changes, can you check please?